src/__init__.py

```diff
@@ -0,0 +1,40 @@
+import asyncio
```

Comment:
why is your main code in a __init__.py file?
always have a main.py file with the main method that starts the program.
__init__ files have a specific purpose, read about it online.